statsmodels/genmod/generalized_linear_model.py

```diff
@@ -494,7 +515,9 @@ def score(self, params, scale=None):
             the sum of `score_obs`
 
         """
-        return self.score_obs(params, scale=scale).sum(0)
+        # internal usage: score should not call score_obs so that subclasses
+        # can override them independently, see GLMPenalized in @4576
```

Comment:
I don't understand this comment.  Why would having the base class `score` call `score_obs` prevent subclasses from overriding them indepdendently?

Reply:
typo `#4576` not `@`

explanation is in comments of this PR, 
this avoids double modification/penalization of score